### PR TITLE
Fix atomic URL

### LINKED ISSUE #2767: AtomicLineList parsing issue
Hi Astroquery, I'm getting the following error with the AtomicLineList today, and not sure how to proceed. It was working fine on Tuesday. Could it be related to yesterday's update for the CDMS table? 
```
from astropy import units as u
from astropy.units import Quantity
from astroquery.atomic import AtomicLineList
AtomicLineList.query_object(wavelength_range=(0.5 * u.um, 1 * u.um), wavelength_type='vacuum',element_spectrum='H')
```
```
/Users/mshahbandeh/miniconda3/envs/June/lib/python3.11/site-packages/astroquery/atomic/core.py:309: GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("html5lib"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 309 of the file /Users/mshahbandeh/miniconda3/envs/June/lib/python3.11/site-packages/astroquery/atomic/core.py. To get rid of this warning, pass the additional argument 'features="html5lib"' to the BeautifulSoup constructor.

  bs = BeautifulSoup(response.text, 'html5')
/Users/mshahbandeh/miniconda3/envs/June/lib/python3.11/site-packages/astroquery/atomic/core.py:249: GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("html5lib"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 249 of the file /Users/mshahbandeh/miniconda3/envs/June/lib/python3.11/site-packages/astroquery/atomic/core.py. To get rid of this warning, pass the additional argument 'features="html5lib"' to the BeautifulSoup constructor.

  data = StringIO(BeautifulSoup(response.text, 'html5').find('pre').text.strip())
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[1], line 4
      2 from astropy.units import Quantity
      3 from astroquery.atomic import AtomicLineList
----> 4 AtomicLineList.query_object(wavelength_range=(0.5 * u.um, 1 * u.um), wavelength_type='vacuum',element_spectrum='H')

File ~/miniconda3/envs/June/lib/python3.11/site-packages/astroquery/atomic/core.py:165, in AtomicLineListClass.query_object(self, wavelength_range, wavelength_type, wavelength_accuracy, element_spectrum, minimal_abundance, depl_factor, lower_level_energy_range, upper_level_energy_range, nmax, multiplet, transitions, show_fine_structure, show_auto_ionizing_transitions, output_columns, cache, get_query_payload)
    163 if get_query_payload:
    164     return input
--> 165 table = self._parse_result(response)
    166 return table

File ~/miniconda3/envs/June/lib/python3.11/site-packages/astroquery/atomic/core.py:249, in AtomicLineListClass._parse_result(self, response)
    248 def _parse_result(self, response):
--> 249     data = StringIO(BeautifulSoup(response.text, 'html5').find('pre').text.strip())
    250     # `header` is e.g.:
    251     # "u'-LAMBDA-VAC-ANG-|-SPECTRUM--|TT|--------TERM---------|---J-J---|----LEVEL-ENERGY--CM-1----'"
    252     # `colnames` is then
    253     # [u'LAMBDA VAC ANG', u'SPECTRUM', u'TT', u'TERM', u'J J',
    254     #  u'LEVEL ENERGY  CM 1']
    256     header = data.readline().strip().strip('|')

AttributeError: 'NoneType' object has no attribute 'text'
```

### PR DESCRIPTION
**Pull request by keflavich**
Fix #2767.

Includes slightly improved error handling for a new known error message.

### CODE REVIEW

File: astroquery/atomic/core.py [hunk 1]
```diff
@@ -248,6 +248,10 @@ def query_object_async(self, *, wavelength_range=None, wavelength_type='', wavel
         return response
 
     def _parse_result(self, response):
+
+        if 'ERROR: request form contains no information' in response.text:
```
Comment (bsipocz) [inline astroquery/atomic/core.py:252]: Could we make it more generic?
Maybe either check only for `'ERROR'`, or split the data assignment below, and check for `'ERROR'` in response text if `BeautifulSoup(response.text, features='html5lib').find('pre')` is None before processing it further?

**Review by bsipocz — COMMENTED** [review]
Do add a changelog entry too, please.

File: astroquery/atomic/core.py (hunk 1 above)
Comment (bsipocz) [reply astroquery/atomic/core.py:252]: Oh, you did both. Thanks!

Comment (keflavich) [reply astroquery/atomic/core.py:252]: I added after your suggestion.

**Comment by keflavich** [comment]
Looks like this is good to merge - want me to squash first?

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks @keflavich!

**Pull request merged**